### PR TITLE
Fc value

### PR DESCRIPTION
**Pull request by MidnightBlackHorse17**

### CODE REVIEW

**Comment by rastguelenianrn** [comment]
Looks pretty good, it looks like there is alot of commits (i have the same problem too, so no worries), so it was kinda hard to find the final product of your function in CardGames.py, but i believe I found it and it looks correct. One quick question, were you ever able to figure out how to determine whether an ace is 1 or 11?

**Comment by rastguelenianrn** [comment]
> Looks pretty good, it looks like there is alot of commits (i have the same problem too, so no worries), so it was kinda hard to find the final product of your function in CardGames.py, but i believe I found it and it looks correct. One quick question, were you ever able to figure out how to determine whether an ace is 1 or 11?

Edit: just realized Jayden's code handles ace values, my bad!

**Comment by MidnightBlackHorse17** [comment]
Thanks for your feedback Ryan! Yeah, I was having some issues with GitHub, so there are a lot of commits..

File: source/CardGames.py [hunk 1]
```diff
@@ -32,6 +32,10 @@ def __eq__(self, other):
     return self.suit == other.suit and \
       self.value == other.value
 
+  def setValue(self):
+    if self.value > 10:
+      self.value = 10
+    return self.value
```
Comment (Jaydenmd) [inline source/CardGames.py:39]: Hi Joanna! Sorry I know I'm not assigned to comment on yours, but you are the next person in line for me to comment on.ut your code looks good! I'm nit sure if there is a simpler way to set the value =10

**Comment by psk44** [comment]
I know I'm very late but everything looks good with the code and the test! Good job!

**Pull request merged**